src/middlewared/middlewared/plugins/smb_/registry_share.py

```diff
@@ -66,7 +66,7 @@ async def netconf(self, **kwargs):
 
     @private
     async def reg_listshares(self):
-        return (await self.netconf(action='listshares')).splitlines()
+        return (await self.netconf(action='listshares')).splitlines()[1:]
```

Comment:
Do you mean just `splitlines()`? Doing it this way will return an empty list if there is only 1 share added.